test/CMakeLists.txt

```diff
@@ -378,6 +378,9 @@ set (H5_TESTS
     cmpd_dtransform
     event_set
 )
+if (HDF5_BUILD_UTILS)
+  set (H5_TESTS ${H5_TESTS} mirror_vfd)
+endif ()
```

Comment:
Why is this test only built based on the HDF5_BUILD_UTILS condition now? From a quick glance it didn't look like it was using anything from the utils dir.

Reply:
mirror_vfd needs the mirror_server in utils right?